source/_integrations/bang_olufsen.markdown

```diff
@@ -150,6 +150,64 @@ data:
   media_content_id: 1234567890
 ```
 
+##### Overlay
+
+Interrupts currently playing media to play an audio message.
+
+To use the Bang & Olufsen Cloud TTS, use `overlay_tts` as the `media_content_type` and enter a message into the `media_content_id` field.
+Bang & Olufsen Cloud TTS messages are limited to 100 unique messages a day and are cached for 24 hours.
+
+Extra keys available:
+
+| Service data attribute    | Optional | Description                                                      |
+| ------------------------- | -------- | ---------------------------------------------------------------- |
+| `overlay_absolute_volume` | yes      | Specify an absolute volume for the overlay.                      |
+| `overlay_offset_volume`   | yes      | Specify a volume offset to be added to the current volume level. |
+| `overlay_tts_language`    | yes      | Specify the language used for text-to-speech.                                            |
```

Comment:
Would it make sense to mention what type of code is used here to specify the language? Is there a default, en-us?